From 6e602bf1ff591c046145d20cd92a1ec148c50938 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 7 Jan 2025 13:18:16 -0800
Subject: [PATCH 1/5] stateless services file path fix

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 src/net/parse_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/net/parse_test.go b/src/net/parse_test.go
index e58d954..34edaa3 100644
--- a/src/net/parse_test.go
+++ b/src/net/parse_test.go
@@ -18,7 +18,7 @@ func TestReadLine(t *testing.T) {
 	case "android", "plan9", "windows", "wasip1":
 		t.Skipf("not supported on %s", runtime.GOOS)
 	}
-	filename := "/etc/services" // a nice big file
+	filename := "/usr/share/defaults/etc/services" // a nice big file
 
 	fd, err := os.Open(filename)
 	if err != nil {
-- 
2.47.1

