From 074029ad9acbd5dc18047eddb3d9e03813ee8be1 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 26 Aug 2022 14:11:51 -0700
Subject: [PATCH] Don't use ntq

---
 src/runtime/memclr_amd64.s | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/runtime/memclr_amd64.s b/src/runtime/memclr_amd64.s
index 19bfa6f..d3ffec4 100644
--- a/src/runtime/memclr_amd64.s
+++ b/src/runtime/memclr_amd64.s
@@ -86,8 +86,6 @@ loop_preheader_avx2:
 	// For smaller sizes MOVNTDQ may be faster or slower depending on hardware.
 	// For larger sizes it is always faster, even on dual Xeons with 30M cache.
 	// TODO take into account actual LLC size. E. g. glibc uses LLC size/2.
-	CMPQ    BX, $0x2000000
-	JAE	loop_preheader_avx2_huge
 
 loop_avx2:
 	VMOVDQU	Y0, 0(DI)
-- 
2.37.2

