From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
Date: Sat, 16 Sep 2017 02:20:20 +0000
Subject: [PATCH] Prepend Clear Linux trust store location.

---
 src/crypto/x509/root_linux.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/crypto/x509/root_linux.go b/src/crypto/x509/root_linux.go
index ad6ce5cae79d..23315c23f4d4 100644
--- a/src/crypto/x509/root_linux.go
+++ b/src/crypto/x509/root_linux.go
@@ -16,6 +16,7 @@ var certFiles = []string{
 
 // Possible directories with certificate files; all will be read.
 var certDirectories = []string{
+	"/var/cache/ca-certs/anchors",  // Clear Linux
 	"/etc/ssl/certs",               // SLES10/SLES11, https://golang.org/issue/12139
 	"/etc/pki/tls/certs",           // Fedora/RHEL
 	"/system/etc/security/cacerts", // Android
